src/pages/Causes/Signup.vue

```diff
@@ -133,12 +133,11 @@ export default {
 			const hasInactiveCauseSubscription = causesSubscriptions.find(
 				subscription => !subscription.enabled
 			);
-
 			return !hasActiveCauseSubscription
 				&& !this.isMonthlyGoodSubscriber
 				&& !this.hasAutoDeposits
 				&& !this.hasLegacySubscription
-				&& (!this.hasMadeLoan && !hasInactiveCauseSubscription);
+				&& (!this.hasMadeLoan || hasInactiveCauseSubscription);
```

Comment:
I believe this is correct. If has not made a loan or has inactive subscription they should be eligible

Reply:
makes sense